server/src/live-pricing.js

```diff
@@ -17,14 +17,9 @@ const formatParams = params => querystring.stringify({
   country: 'UK',
   currency: 'GBP',
   locale: 'en-GB',
-  locationschema: 'Sky',
+  locationSchema: 'Sky',
   apiKey: config.apiKey,
-  adults: params.adults,
-  cabinclass: params.class,
-  destinationplace: params.toPlace,
-  inbounddate: params.toDate,
-  originplace: params.fromPlace,
-  outbounddate: params.fromDate,
+  ...params,
```

Comment:
Since we are no longer specifying in code what params are required, could this make it a bit trickier for candidates to find all the params they're supposed to pass in?

Reply:
I guess maybe it could yeah, although I think it's better than the current state, as currently we have a comment telling candidates to look at the API docs, but the only way they know what to pass in is by looking in `live-pricing.js`. This way, our comment is valid and it's up to them to check the API docs

Reply:
Fair enough! I think it's reasonable to expect them to check and understand the API docs :D